general-concepts/slotting/text.xml

```diff
@@ -45,6 +52,102 @@ To <c>DEPEND</c> upon a package in a specific slot, refer to
 </p>
 </body>
 
+<section>
+<title>Considerations for slotting</title>
+<body>
+
+<p>
+Slotting is a powerful tool and as such, is easily abused. It's critical
+that when slotting, package maintainers carefully consider the impact
+on <e>reverse dependencies</e>.
+</p>
+
+<p>
+Do not be fooled by the ease in adapting the package itself to slotting:
+slotting the package itself is always the easiest part.
+</p>
+
+<p>
+Before proceeding at all, consider the impact on the wider ecosystem
+beyond Gentoo. Especially examine other distributions and their handling
+of the problem in question. Gentoo cannot unilaterally move software to
```

Comment:
Well, technically speaking it can. I think the point's more like "it's not always a good idea".